pandas/tests/extension/test_categorical.py

```diff
@@ -175,9 +175,8 @@ def test_combine_add(self, data_repeated):
     def test_fillna_length_mismatch(self, data_missing):
         super().test_fillna_length_mismatch(data_missing)
 
-    def test_searchsorted(self, data_for_sorting):
-        if not data_for_sorting.ordered:
-            raise pytest.skip(reason="searchsorted requires ordered data.")
+    def test_searchsorted(self, data_for_sorting, as_series):
+        super().test_searchsorted(data_for_sorting, as_series)
```

Comment:
could remove entirely if no need to override? but maybe just update to xfail depending on base test change discussion.

Reply:
Yes, can be removed entirely